MTFO/Managers/CustomPuzzleManager.cs

```diff
@@ -0,0 +1,150 @@
+﻿using ChainedPuzzles;
+using UnityEngine;
+using static MTFO.Custom.CustomBioScan;
+using MTFO.Custom;
+using MTFO.Utilities;
+using System;
+using System.Collections.Generic;
+
+namespace MTFO.Managers
+{
+    /// <summary>
+    /// Handles all custom puzzle related stuff so it
+    /// isn't in a single patch like c'mon Dak you
+    /// had one job.
```

Comment:
?

Reply:
3am dak moment

Reply:
The only job of this manager class tbh is so I don't reference Patch_PuzzleManager because that would be cringe